drivers/python/rethinkdb/_import.py

```diff
@@ -793,7 +803,9 @@ def import_directory(options):
 
     spawn_import_clients(options, files_info)
 
-def table_check(progress, conn, db, table, pkey, force):
+def table_check(progress, conn, db, table, create_args, force):
+    pkey = create_args["primary_key"]
```

Comment:
This will crash if `primary_key` isn't present in `create_args`, which happens if it is not specified on the command-line.  Line 820 should probably just set `pkey` directly.  The two checks comparing `pkey` to `None`, should probably use `'primary_key' in create_args`.